WordPress/Classes/Utility/Analytics/WPAnalyticsEvent.swift

```diff
@@ -836,6 +842,16 @@ import Foundation
             return "my_site_dashboard_card_shown"
         case .dashboardCardItemTapped:
             return "my_site_dashboard_card_item_tapped"
+        case .mySiteTabTapped:
+            return "my_site_tab_tapped"
+        case .mySiteSiteMenuShown:
+            return "my_site_site_menu_show"
```

Comment:
Typo -- I'll validate this event after this is fixed
```suggestion
            return "my_site_site_menu_shown"
```

Reply:
Fixed in 052ac47e8dfc391d1235e17ab6d7fbfea9eaedc1